base/exports.jl

```diff
@@ -23,6 +23,7 @@ export
     Docs,
     Markdown,
     Threads,
+    Iterators,
```

Comment:
No. `using` can't see `Base.Iterators` unless you do e.g. `import Base.Iterators`.